features/dd-sdk-android-flags/src/main/kotlin/com/datadog/android/flags/internal/evaluation/EvaluationsManager.kt

```diff
@@ -76,18 +78,21 @@ internal class EvaluationsManager(
                 )
 
                 flagStateManager.updateState(FlagsClientState.Ready)
+                callback?.onSuccess()
             } else {
                 internalLogger.log(
                     InternalLogger.Level.WARN,
                     InternalLogger.Target.USER,
                     { NETWORK_REQUEST_FAILED_MESSAGE }
                 )
 
+                val throwable = Throwable(NETWORK_REQUEST_FAILED_MESSAGE)
```

Comment:
minor: it is better to avoid creating generic `Throwable`. Probably here it is better to create some `NetworkCallFailedException: RuntimeException` and pass it instead.